spec/models/entity_spec.rb

```diff
@@ -0,0 +1,10 @@
+# spec/models/entity_spec.rb
+require 'rails_helper'
+
+RSpec.describe Entity, type: :model do
+  it { should validate_presence_of(:name) }
+  it { should validate_presence_of(:amount) }
+  it { should validate_numericality_of(:amount) }
+  it { should belong_to(:author).class_name('User') }
```

Comment:
- [ ] Just like the previous reviewer rightly pointed out, this particular test is still failing. Kindly look into it and ensure that all your tests are green ✔️ 
![image](https://github.com/rudicarrilloypr/actions-app/assets/46839548/79a2b53e-9f51-40d7-b95d-48383835ee11)

Reply:
@codepantha thats weird, my tests are passing perfectly:
<img width="627" alt="Captura de pantalla 2023-10-27 a la(s) 9 14 46" src="https://github.com/rudicarrilloypr/actions-app/assets/125106965/7b621fa5-05ab-40e9-9642-cdf6d2f507c7">
<img width="627" alt="Captura de pantalla 2023-10-27 a la(s) 9 16 17" src="https://github.com/rudicarrilloypr/actions-app/assets/125106965/d2e17d9b-1f6f-48df-9252-4b5f2155e764">